__tests__/Unit/Components/Tasks/ProgressCard.test.tsx

```diff
@@ -55,12 +55,18 @@ describe('ProgressCard Component', () => {
             </Provider>
         );
         const progressArr = container.querySelectorAll(
-            '[data-testid="progress-item"]'
```

Comment:
progress-item data-testid was present in the below list and it is deleted as we are not using it anymore. latest dates are present in tooltip. So tooltip is used to query the elements and text contend is also changed to new format.
```
               <li
                    onClick={openDetails}
                    className={styles['list-item']}
                    data-testid="progress-item"
                >
                    {getDateFromTimestamp(data.date)}
                </li>
```